WalkingChallenge/Challenge/Challenge.swift

```diff
@@ -31,4 +31,161 @@ import UIKit
 import Foundation
 
 class Challenge: UIViewController {
+  private let lblTitle: UILabel = UILabel(typography: .headerTitle)
+  private let btnSettings: UIButton = UIButton(type: .system)
+  private let tblCards: UITableView = UITableView(frame: .zero)
+
+  private var arrCards: [JourneyCardMiles] = []
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    self.navigationController?.navigationBar.barTintColor = #colorLiteral(red: 0.9647058823, green: 0.9725490196, blue: 0.9803921568, alpha: 1.0000000000)
```

Comment:
Is the color here not one of the standard colors from the AKF palette?  If so, why not?  If it is, why not use the color constants?